resources/metadata/spotBugs.yaml

```diff
@@ -0,0 +1,33 @@
+metadata:
+  name: checkSpotBugs
```

Comment:
we should reconsider the step naming.
Does this really fit to our general step naming pattern?